README.md

```diff
@@ -1,3 +1,39 @@
 # SnowMan
 
-This is a console Swift application. It plays like Hangman but without the gore.
+Snowman is a Swift console application. The object of the game is to guess the given word or phrase before the snowman "melts". With each incorrect guess, the snowman will lose one of its snowman features until he is but a puddle. 
+
+## Requirements
+
+[Swift](https://swift.org/) 
+[Swift Package Manager](https://swift.org/package-manager/)
+
+## Gameplay
+
+To play the game... TBD
+
+## Running the Application
```

Comment:
`Gameplay` and `Running the Application` seem redundant. I think that running the application is going to fall into the scope of gameplay.